### PR TITLE
Catalogue item system integration tests #334

### LINKED ISSUE #334: Catalogue items system integration tests

### PR DESCRIPTION
**Pull request by joshuadkitenge**
## Description

catalogue item systems tests 
recognizing e2e with api 
add aria-label to table row 

## Testing instructions

Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] {more steps here}

## Agile board tracking

closes #334

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "e2e:api": "cross-env USE_API_SETTINGS=true start-server-and-test e2e:serve http://localhost:3000 cy:run:api",
     "cy:open": "cross-env TZ=UTC cypress open",
     "cy:run": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_mock_data/*.cy.ts\"",
-    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/*.cy.ts\""
+    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/\""
```
Comment (joelvdavies) [inline package.json:52]: Why did this need changing? I thought it only specified the files to run?

**Review by joelvdavies — COMMENTED** [review]
Single comment, ran well locally.

File: package.json (hunk 1 above)
Comment (joelvdavies) [reply package.json:52]: Ah I see, can you use `./cypress/e2e/with_api/**/*.cy.ts` instead so it doesn't attempt to look at the functions files, it looks like it finds the subdirectories and `app.cy.ts` as well when I tested it.

Comment (joshuadkitenge) [inline package.json:52]:
```suggestion
    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/**/*.cy.ts/\""
```

File: package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
     "e2e:api": "cross-env USE_API_SETTINGS=true start-server-and-test e2e:serve http://localhost:3000 cy:run:api",
     "cy:open": "cross-env TZ=UTC cypress open",
     "cy:run": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_mock_data/*.cy.ts\"",
-    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/*.cy.ts\""
+    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/**/*.cy.ts/\""
```
Comment (joshuadkitenge) [inline package.json:52]:
```suggestion
    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/**/*.cy.ts\""
```

**Review by joelvdavies — APPROVED** [review]

**Pull request merged**